system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -150,6 +150,10 @@ $half-base-gutter: $base-gutter / 2;
     border-radius: $top-sites-border-radius;
     box-shadow: inset $inner-box-shadow;
     position: absolute;
+
+    .dark-theme & {
+      box-shadow: none;
+    }
```

Comment:
Alternatively, I could do this below which is probably more consistent with the rest of our theming? Thoughts?

```
diff --git a/system-addon/content-src/components/TopSites/_TopSites.scss b/system-addon/content-src/components/TopSites/_TopSites.scss
index 81c3859d..aed108e2 100644
--- a/system-addon/content-src/components/TopSites/_TopSites.scss
+++ b/system-addon/content-src/components/TopSites/_TopSites.scss
@@ -148,12 +148,8 @@ $half-base-gutter: $base-gutter / 2;
     background-position: center center;
     background-repeat: no-repeat;
     border-radius: $top-sites-border-radius;
-    box-shadow: inset $inner-box-shadow;
+    box-shadow: var(--newtab-topsites-icon-shadow);
     position: absolute;
-
-    .dark-theme & {
-      box-shadow: none;
-    }
   }

   .rich-icon {
diff --git a/system-addon/content-src/styles/_theme.scss b/system-addon/content-src/styles/_theme.scss
index 735ed60e..2362e304 100644
--- a/system-addon/content-src/styles/_theme.scss
+++ b/system-addon/content-src/styles/_theme.scss
@@ -52,6 +52,7 @@ body {

   // Top Sites
   --newtab-topsites-background-color: $white;
+  --newtab-topsites-icon-shadow: 0 0 0 1px var(--newtab-inner-box-shadow-color);
   --newtab-topsites-label-color: inherit;

   // Cards
@@ -106,6 +107,7 @@ body {

   // Top Sites
   --newtab-topsites-background-color: $grey-70;
+  --newtab-topsites-icon-shadow: none;
   --newtab-topsites-label-color: $grey-10-80;

   // Cards
```

Reply:
I think I'm leaning to keeping `.dark-theme * {}` stuff contained in _theme.scss 🤔

Reply:
Nod. Let's do the css variable for the shadow